### ISSUE TITLE
Decide on versioning scheme

### ISSUE THREAD

**Issue by cweagans** [universal-ctags/ctags#661]
@masatake brought this up here: https://github.com/universal-ctags/ctags/issues/652#issuecomment-153411749

I prefer using [semver](http://semver.org/) and starting with 1.0.0 since Universal Ctags != Exuberant Ctags. @masatake prefers 6.0.

When this discussion is resolved:
- [ ] Update the name of the "Initial release" milestone to whatever is decided here as the first "stable" release version

I'm also in favor of having an earlier non-stable release (just for the sake of putting a line in the sand that people working on packaging can reference), perhaps 0.1.0?

cc @universal-ctags/admins @universal-ctags/developers

**Comment by masatake** [comment]
@cweagans, I follow your proposal: 1.0.0 and 0.1.0.

**Comment by cweagans** [comment]
I propose we do the following:

* Rename the initial release milestone to "1.0.0"
* When all the issues in the 1.0.0 milestone are closed, we tag 1.0.0-rc1
* After a month, if there are no more critical, release-blocking issues, we tag 1.0.0. If there are, we fix them, and tag 1.0.0-rc2, rinse, and repeat until we can get through an entire month without release blocking issues.

@universal-ctags/admins any objections to this ^?

**Comment by ffes** [comment]
Sounds good to me.

Just think that 0.1.0 is a bit low. 0.9.0 sounds more like we are working towards a proper release.

**Comment by cweagans** [comment]
@ffes I don't think we should do a 0.1.0. Let's just do 1.0.0 with release candidates before the final release in case anything comes up.

**Comment by blueyed** [comment]
:+1:

**Comment by Wilfred** [comment]
Could we tag a 0.9.0 or a release candidate at this point? There are [19 issues](https://github.com/universal-ctags/ctags/milestones/1.0.0) outstanding for 1.0.0, but the code is very useful today.

**Comment by blueyed** [comment]
@Wilfred 
I think https://github.com/universal-ctags/ctags/issues/781 is a major blocker, but has a PR (although considered to be unclean?!).

**Comment by romgrk** [comment]
I would recommend using 6.0, this already causes some issues, eg https://github.com/xolox/vim-easytags/pull/133. Bump btw.

**Comment by masatake** [comment]
Though I've got many valuable comments, I will use "6.0.0" as the initial tarbal'ed release.
Till these days, I have not had any merit for me to make a release.

However, I guess the versioning may help the development of the citre.
My daily work heavily depends on citre. So I decided to release 6.0.0 with a tarball.

I will release 6.x.0 or 7.0.0 if ctags provides a new feature that has a big impact on citre.
e.g. generic reference tags.

Thank you for the comments. I decides to be selfish for working on ctags longer time.
